packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { openmrsFetch } from '@openmrs/esm-framework';
+import AppointmentsMetrics from './appointments-metrics.component';
+import { mockAppointmentMetrics } from '../../../../__mocks__/appointments.mock';
+
+const mockedOpenmrsFetch = openmrsFetch as jest.Mock;
+
+jest.mock('../hooks/useClinicalMetrics', () => {
+  const originalModule = jest.requireActual('../hooks/useClinicalMetrics.tsx');
+
+  return {
+    ...originalModule,
+    useClinicalMetrics: jest.fn().mockImplementation(() => ({
+      totalAppointments: mockAppointmentMetrics.totalAppointments,
+      highestServiceLoad: mockAppointmentMetrics.highestServiceLoad,
+      isLoading: mockAppointmentMetrics.isLoading,
+      error: mockAppointmentMetrics.error,
+    })),
+  };
+});
+
+jest.mock('@openmrs/esm-framework', () => {
+  const originalModule = jest.requireActual('@openmrs/esm-framework');
+
+  return {
+    ...originalModule,
+  };
+});
+
+describe('Appointment metrics', () => {
+  it('renders metrics from appointment list', async () => {
```

Comment:
Also appears like we're not doing anything asynchronous in the test body, so we wouldn't need this either.